ui-bookly/components/OrganisationCard.tsx

```diff
@@ -0,0 +1,13 @@
+import { Grid, Card, TextField, Container } from '@mui/material';
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+
+const OrganisationCard = ( props: any ) => {
```

Comment:
A few things here.

### Destructuring the props and using types
A better way to pass props is like this
```
const OrganisationCard = ( { organisation } ) => {
```

by going {} inside, you're deconstructing the props object into the attributes you expect, so it's clear to whoever is reading it what you're passing to the component.

Say we have a prop like this

```
<OrganisationCard name={name} abn={abn}/>
```

we could then instantiate it like this

```
const OrganisationCard = ( { name, abn } ) => {
```

Another thing is since you're using TS, you might as well give proper types as well. Generally you don't want to use `any` except for maybe some testing. 

It would like like this for example:

```
type Organisation {
    name: string;
    abn: string;
}

const OrganisationCard = ( { name, abn } : Organisation ) => {
```

### Passing it only what you need vs entire organisation object

Whether you want to pass the whole org or just the things you need is a matter of preference. If you only use the name it might be simpler to just pass the name. If you wanted to do it by passing the whole org object you'd do it like

```
type Organisation {
    name: string;
    abn: string;
}

type Props { 
    organisation: Organisation
}

const OrganisationCard = ( { organisation } : Props ) => {
```

Reply:
Very helpful, ty!